static/js/components/target.vue

```diff
@@ -185,6 +194,16 @@ export default {
         } else {
           that.lookupText = 'Could not find any matching objects';
           that.lookupFail = true;
+          for (var x in that.ns_target_params){
```

Comment:
Can you explain the need for this chunk?

Reply:
It's intended to clear the fields if the user searches for something that isn't there, for the use case where a user looks for something that exists, the fields fill, and they choose to look for something else that doesn't exist. A failed lookup will clear the fields.

Reply:
Ok, the code is repeated here: https://github.com/LCOGT/valhalla/blob/a31cf78898f8aacccfc9c68686dd612bd69cf685/static/js/components/target.vue#L237
Any chance you can factor the "clearing" code out to keep it DRY?

Reply:
Yeah, I can likely extract it into a method so it's cleaner.